### ISSUE TITLE
Switch the diving method

### ISSUE THREAD

**Issue by ghostdevv** [jonsuh/hamburgers#104]
Dart sass 2.0 is removing the `/` method of diving things, source: https://sass-lang.com/documentation/breaking-changes/slash-div
This is causing a few deprecation errors with the times that things are divided with hamburgers, this should be updated

**Comment by cedricbommart** [comment]
I agree I had the issue recently. Please use the sass-migrator to fix the issues in seconds : https://sass-lang.com/documentation/cli/migrator

```bash
$ npm i -g sass-migrator
$ sass-migrator division --migrate-deps _sass/hamburgers/base.scss
$ ...
```

**Comment by lewebsimple** [comment]
I don't want to run migration commands each time I install an npm package ... you can use my fork at https://github.com/lewebsimple/hamburgers/ until @jonsuh merges the PR above.

**Comment by ghostdevv** [comment]
@cedricbommart Not a bad solution, I do agree with @lewebsimple that I don't want to have to run this but if I have to I have to - Hopefully @jonsuh will merge the pr made.

**Comment by cedricbommart** [comment]
Yep, sorry @lewebsimple @ghostdevv , I haven't mention it but my  message was more adressed to @jonsuh 🙏❤️

**Comment by ghostdevv** [comment]
@cedricbommart alright, might still be useful for those who don't want to use a fork, and instead patch the package.

**Comment by jonsuh** [comment]
Resolved via https://github.com/jonsuh/hamburgers/pull/106
